Pod/Classes/SideMenuNavigationController.swift

```diff
@@ -264,8 +264,9 @@ open class SideMenuNavigationController: UINavigationController {
     override open func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
         super.viewWillTransition(to: size, with: coordinator)
         
-        // Don't bother resizing if the view isn't visible
-        guard let transitionController = transitionController, !view.isHidden else { return }
+      // Don't bother resizing if the view isn't visible or in background
+      guard let transitionController = transitionController, !view.isHidden,
+            UIApplication.shared.applicationState != UIApplication.State.background else { return }
```

Comment:
This change fix the issue only if the device is not rotated, but it's a good approach to fix the rotation issue. 
Thanks

Reply:
Adding the inactive case to your fix I was able to fix the weird transition when rotates the device in background too.

guard let transitionController = transitionController, !view.isHidden, 
           UIApplication.shared.applicationState != UIApplication.State.background, 
           UIApplication.shared.applicationState != UIApplication.State.inactive else { return }